### PR TITLE
squashing deprecation warnings

### PR DESCRIPTION
**Pull request by bollwyvl**
Had to rebuild my nbviewer environment, and started seeing a bunch of deprecation warnings on run.

These may indeed break the 2.7 build, but we'll see what travis says :)

### CODE REVIEW

File: nbviewer/utils.py [hunk 1]
```diff
@@ -5,7 +5,14 @@
 #  the file COPYING, distributed as part of this software.
 #-----------------------------------------------------------------------------
 
-import base64
+# https://docs.python.org/3.1/library/base64.html#base64.decodestring
+try:
+    from base64 import encodebytes as encodestring
+    from base64 import decodebytes as decodestring
+except ImportError:
+    from base64 import encodestring
```
Comment (minrk) [inline nbviewer/utils.py]: probably makes more sense to import `deprecated as new` than `new as deprecated`, and update below.

**Comment by bollwyvl** [comment]
Updated that. Back to test roulette!

I'm also trying to get my docker setup running again; always good to have a look at that :)

**Comment by rgbkrk** [comment]
:smile:

**Comment by bollwyvl** [comment]
destroy all spurious output!

**Pull request merged**